src/applications/appeals/10182/tests/10182-keyboard-only.cypress.spec.js

```diff
@@ -46,7 +46,7 @@ describe('Notice of Disagreement keyboard only navigation', () => {
 
       // Homelessness radios
       cy.url().should('include', chapters.infoPages.pages.homeless.path);
-      cy.tabToElement('[name="root_homeless"]');
+      cy.tabToElement('input[name="root_homeless"]');
```

Comment:
Was this failing before? Or is this because of the update in `homeless.js`?

Reply:
There is a `va-radio[name="root_homeless"]` and a `va-radio-option[name="root_homeless"]` that also match; so I added `input` in front.